src/Donations/Micros/PaymentStatus/ImageComponent.tsx

```
@@ -1,7 +1,7 @@
 import React, { ReactElement, RefObject } from "react";
 import { useTranslation } from "next-i18next";
 import { getFormattedNumber } from "src/Utils/getFormattedNumber";
-import getFormatedCurrency from "src/Utils/getFormattedCurrency";
+import getFormattedCurrency from "src/Utils/getFormattedCurrency";
 import getFormattedCurrency from "src/Utils/getFormattedCurrency";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getFormattedCurrency'. Consider to delete it or rename it.

'getFormattedCurrency' is defined here:


(lint/suspicious/noRedeclare)